apps/api/src/server.ts

```diff
@@ -38,17 +39,21 @@ const API_DOCS_PATH = `api-docs`;
 const startServer = async () => {
 	const app = express();
 
+	// app.use(ExpressLogger({ logger, excludeURLs: ['/auth/token'] }));
 	app.use(ExpressLogger({ logger }));
 
+	app.use(sessionMiddleware);
```

Comment:
Adding session middleware here. It is defined in `./session/sessionMiddleware.ts` and uses the valkeyClient defined in `./session/valkeyClient.ts`